Signal/src/view controllers/CallViewController.swift

```diff
@@ -287,21 +519,25 @@ class CallViewController: UIViewController {
         self.dismiss(animated: true)
     }
 
-    @IBAction func didPressMute(sender muteButton: UIButton) {
+    func didPressMute(sender muteButton: UIButton) {
         Logger.info("\(TAG) called \(#function)")
         muteButton.isSelected = !muteButton.isSelected
         CallService.signalingQueue.async {
             self.callService.handleToggledMute(isMuted: muteButton.isSelected)
         }
     }
 
-    @IBAction func didPressSpeakerphone(sender speakerphoneButton: UIButton) {
+    func didPressSpeakerphone(sender speakerphoneButton: UIButton) {
         Logger.info("\(TAG) called \(#function)")
         speakerphoneButton.isSelected = !speakerphoneButton.isSelected
         audioService.isSpeakerphoneEnabled = speakerphoneButton.isSelected
     }
 
-    @IBAction func didPressAnswerCall(sender: UIButton) {
+    func didPressTextMessage(sender speakerphoneButton: UIButton) {
+        Logger.info("\(TAG) called \(#function)")
```

Comment:
My inclination is to eventually make a UIButton category/extension so that we can gather concerns and define button behavior where the button is created.